packages/guideline-blocks-settings/src/index.ts

```diff
@@ -27,6 +26,7 @@ import type {
     TextareaBlock as TextareaBlockSidebarSettings,
     ValueOrPromisedValue as ValueOrPromisedValueSidebarSettings,
 } from '@frontify/sidebar-settings';
+import type { FC } from 'react';
```

Comment:
Please revert the change and set your IDE to not change these lines, it is noise and really not useful.